web/js/map-OpenLayers.js

```diff
@@ -144,6 +173,25 @@ function fixmystreet_onload() {
             backgroundHeight: 40,
             backgroundXOffset: -10,
             backgroundYOffset: -35
+        },
+        'small': {
+            externalGraphic: fixmystreet.pin_prefix + "pin-${colour}-small.png",
+            graphicWidth: 24,
+            graphicHeight: 32,
+            graphicXOffset: -12,
+            graphicYOffset: -32,
```

Comment:
I think the offset numbers need looking at, as the bubbles don't appear in the right places like they do on the normal sized pins.